src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/Delegation/DesignTimeHelperResponseRewriter.cs

```diff
@@ -95,20 +95,21 @@ private class CompletionItemComparer : IEqualityComparer<CompletionItem>
     {
         public static CompletionItemComparer Instance = new();
 
-        public bool Equals(CompletionItem x, CompletionItem y)
+        public bool Equals(CompletionItem? x, CompletionItem? y)
         {
-            if (x is null && y is null)
+            if (x is null)
             {
-                return true;
+                return y is null;
             }
-            else if (x is null || y is null)
+            else if (y is null)
             {
                 return false;
             }
 
-            return x.Label.Equals(y.Label, StringComparison.Ordinal);
+            return x.Label == y.Label;
```

Comment:
Why not `Equals`? I was raised to always use `Equals` with a specific comparer for strings.

Reply:
Calling `Equals` with `StringComparison.Ordinal` is just a slower and wordier way to use `==` for strings. In general, I'm of the opinion that `string.Equals` should be used in specific cases, such as case-insensitive strings.